src/firefly/js/templates/lightcurve/LCPanels.css

```diff
@@ -52,5 +52,5 @@
     background-color: #dcdcdc;
     border: 1px solid #cccccc;
     margin-bottom: 3px;
-    padding: 5px 10px;
+    padding: 5px 10px 10px 5px;
```

Comment:
minor.. adding 10px 5px causes left/right padding mismatch.  I would think you want 10px 10px, instead.